src/target/source/interface_c.cc

```diff
@@ -90,8 +92,12 @@ class InterfaceCNode : public runtime::ModuleNode {
     for (const tir::usmp::AllocatedPoolInfo pool : pools_) {
       String pool_name = pool->pool_info->pool_name;
       Integer pool_size = pool->allocated_size;
-      EmitIntegerValueMacro(code, SanitizeName(pool_name) + " size",
-                            SanitizeName(pool_name) + "_WORKSPACE_POOL_SIZE", pool_size->value);
```

Comment:
This is a breaking change  and I dont see the requirement to do this.
For this PR, my suggestion would be to add _CONSTANT_POOL_SIZE and _CONSTANT_POOL_DATA seperately. 

We can revisit if we want to change this behaviour and consolidate to a single pool size.

Reply:
Amended